src/main/java/com/BrassAmber/ba_bt/entity/hostile/golem/BTGolemEntity.java

```diff
@@ -18,4 +22,10 @@ protected void registerGoals() {
 		
 		this.goalSelector.addGoal(1, new GolemStompAttackGoal(this, 4.0F, 6));
 	}
+
+	/*********************************************************** Properties ********************************************************/
+
```

Comment:
Feels like we need an 'Override' here to make it clear?

Reply:
Done :)